assests/javascript/index.js

```diff
@@ -17,9 +22,9 @@ async function getWeather() {
   }
 }
 
-async function getCityLocation() {
+async function getCityLocation(cityName) {
   const fetchedData = await fetch(
-    `${baseUrl}geo/1.0/direct?q=Rasht,IR&appid=${apiKey}`
+    `${baseUrl}geo/1.0/direct?q=${cityName}&appid=${apiKey}`
```

Comment:
Is this change correct? The previous value was `Rasht,IR` and now it's `rasht`. 
I recommend to use city id if it's available in the api (check the api and see if there is a unique identifier for the cities and if it's possible to fetch the cities by that)

Reply:
you are right,I will fix it